care/facility/migrations/0089_testinglab_squashed_0092_auto_20200414_2205.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 2.2.11 on 2020-04-14 16:36
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import phonenumber_field.modelfields
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'phonenumber_field.modelfields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348729)